config.schema.json

```diff
@@ -55,6 +60,12 @@
             "title": "Show if bin is full as a contact sensor",
             "required": false
             },
+            "dockOnStop": {
```

Comment:
There is some confusion over the negative vs positive naming of this option. Here's it's `dockOnStop` but its value actually means the opposite, leading to the confusing piece of code:

```javascript
this.nodockOnStop = config.dockOnStop
```

I suggest we standardise on `noDockOnStop`.